features/tags_and_wrangling/tag_wrangling_special.feature

```diff
@@ -180,3 +180,16 @@ Feature: Tag Wrangling - special cases
     Then I should see "is not a canonical tag. Please make it canonical before adding synonyms to it."
     When I follow 'Evelyn "Evie" Carnahan'
     Then I should be on the 'Evelyn "Evie" Carnahan' tag edit page
+
+  Scenario: Cannot uncanonize a tag on creation (AO3-6387)
```

Comment:
Independent of what happens above, couldn't this be pulled down into the unit test specs? I tend to only reach for integration tests when absolutely necessary given that they tend to run slower

Reply:
Cannot be a unit if the check remains in the controller (as per the conclusion of the above discussion). But I guess going for a more lightweight controller test for a controller change does make a lot of sense.